test/functional/test_framework/test_node.py

```diff
@@ -170,7 +170,7 @@ def wait_for_rpc_connection(self):
                 self.url = self.rpc.url
                 self.log.debug("RPC successfully started")
                 self.time_to_connect = datetime.now() - start_time
-                return
+                return self.time_to_connect
```

Comment:
This change does not appear to be related to the intention of the PR.

Reply:
Yes, there were some unrelated commits in this PR. I cleaned up the branch log. Now it should be correct